### DISCUSSION TITLE
Package changelog

### DISCUSSION THREAD

**Discussion by HarrySky** [FreeOpcUa/opcua-asyncio#1010] [General]
Hey, thank you for this great package, did you consider creating a changelog? :smile: 

Right now to make sure updates to new version do not break anything you need to go through commits or update `asyncua` dependency and see whether something fails in tests / dev deployment.

Changelog can be in `CHANGELOG.md` file and look like this: https://keepachangelog.com/en/1.0.0/

(you don't necessary need the changelog to start from the first version)

**Comment by oroulet** [comment]
We should have a changelog but this is an open source project with no paid employee. I did not have time to make any changelog, but if you (or others) want to contribute a changelog you are more than welcome.

**Reply by HarrySky** [reply]
Yes, I will create a PR in the near future to create one, starting from previous version
